spid_cie_oidc/onboarding/tests/authn_request_settings.py

```diff
@@ -24,6 +24,52 @@
     "ui_locales": ["codice1", "codice2", "codice3"]
 }
 
+AUTHN_REQUEST_SPID_NO_CLIENT_ID = {
```

Comment:
Please refactor in the following way, I want to save your time and yur brain with the help of the nowadays best practices.

Don't copy/paste, inherit and overload

````
from copy import deepcopy

AUTHN_REQUEST_SPID_NO_CLIENT_ID = deepcopy(AUTHN_REQUEST_SPID)
AUTHN_REQUEST_SPID_NO_CLIENT_ID.pop('client_id')
````
33 lines of code are now useless. Do you agree with me?

Reply:
it would be more easy read the delta of changes without all the previous mouse scolling